includes/class-wc-calypso-bridge-payments.php

```diff
@@ -161,7 +167,7 @@ public function enqueue_payments_welcome_page_scripts() {
 		wp_register_style(
 			'wcpay-welcome-page',
 			plugins_url( '../build/style-index.css', __FILE__ ),
-			// Add any dependencies styles may have, such as wp-components.
+			array(), // Add any dependencies styles may have, such as wp-components.
```

Comment:
Is this right? I think there wasn't an array here in the first place (?)

Reply:
Yes, the change is right.
They had missed the 3rd argument (dependencies) and instead, they had the version as the 3rd argument.

It was coming up from PHPCS, saying that "wp_register_style does not have a version and it will affect browser caching" (or something along these lines :) )

See screenshot

![image](https://user-images.githubusercontent.com/2484390/190361817-c05eaf4e-07d1-4a2d-8c56-f15d63ef9c7d.png)

Reply:
Alright! Nice catch 💪